blocks/api/parser.js

```diff
@@ -192,19 +198,18 @@ export function createBlockWithFallback( blockNode ) {
 
 	attributes = attributes || {};
 
-	// Trim content to avoid creation of intermediary freeform segments
+	// Trim content to avoid creation of intermediary freeform segments.
 	innerHTML = innerHTML.trim();
 
 	// Use type from block content, otherwise find unknown handler.
 	name = name || getUnknownTypeHandlerName();
 
-	// Convert 'core/text' blocks in existing content to the new
-	// 'core/paragraph'.
+	// Convert 'core/text' blocks in existing content to 'core/paragraph'.
 	if ( 'core/text' === name || 'core/cover-text' === name ) {
```

Comment:
It's probably time to remove this old use-case.

Reply:
I was thinking of switching back to text :)